renku/cli/_config.py

```diff
@@ -158,3 +159,48 @@ def find_project_config_path(path=None):
         project_path = get_project_config_path(parent)
         if project_path:
             return project_path
+
+
+@attr.s
+class ConfigManager:
+    """Client for handling global configuration."""
+
+    config_dir = attr.ib(default=default_config_dir(), converter=str)
+    config_name = attr.ib(default='renku.ini', converter=str)
+
+    @property
+    def config_path(self):
+        """Renku config path."""
+        config = Path(self.config_dir)
+        if config.exists() is False:
+            config.mkdir()
+
+        return config / Path(self.config_name)
+
+    def load_config(self):
+        """Loads global configuration object."""
+        config = configparser.ConfigParser()
+        config.read(self.config_path)
+        return config
+
+    def store_config(self, config):
+        """Persists global configuration object."""
+        with open(self.config_path, 'w') as file:
+            config.write(file)
+
+    def load_secret(self, provider):
+        """Load a secret for a external provider."""
+        try:
+            config = self.load_config()
+            return config[provider]['secret']
```

Comment:
maybe `return config[provider].get('secret')` instead of try/except?

Reply:
`config` is not a dict but a config section object. But still, point taken. I've looked it up how to do it there so it should be updated, also added a test for it while at it. Thanks.

Reply:
👍